app/views/admin/tags/new.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class="crayons-card p-6">
-  <h4 class="mb-4">New Tag</h4>
+  <h1 class="crayons-title ml-3 xl:ml-0 mb-3">New Tag</h1>
```

Comment:
The result of this change is 

<img width="500" alt="Screenshot 2023-01-13 at 3 59 08 PM" src="https://user-images.githubusercontent.com/9396084/212298399-e3ce42c9-c25b-4ed6-8d63-25eae2396143.png">

i.e, the `New Tag` text is inside the card.

But if we look at similar other cases like `Create Badge` or `Make a New HTML Variant` the heading is outside the card.
Lets make it consistent.
<img width="300" alt="Screenshot 2023-01-13 at 3 59 02 PM" src="https://user-images.githubusercontent.com/9396084/212298655-2f81520c-68f4-44cd-a879-a751e250c87e.png"> <img width="300" alt="Screenshot 2023-01-13 at 3 59 28 PM" src="https://user-images.githubusercontent.com/9396084/212298679-a51821f9-9b87-4030-a7f8-72475fd5b948.png">

Reply:
I put this in a different comment as well. I think the general way that we're moving towards this is that the header + button should be within the card. I say this because Member Manager was the most recently updated page where we got a design from Anuj and the headers were included in the card. However, I think I'd prefer that we get more specific guidance from @anujbhavsar96 on what the best practice is before making changes like this. Perhaps this is also an issue that we can add to our open source board for community contributions in the future.

Reply:
@Ridhwana, you are right. All the admin feature we have worked on has header + btn inside the card. That gives a better flow of reading and association with the content below.

IMO, it's not a major issue that affects UX, we can probably revisit these whenever we decide to work on those particular pages holistically, which will help us make better decisions at that time.

Reply:
Sounds good.